examples/ecdsa/methods/guest/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "ecdsa-verify"
+version = "0.1.0"
+edition = "2021"
+
+[workspace]
+
+[dependencies]
+risc0-zkvm = { path = "../../../../risc0/zkvm", default-features = false, features = ["std"] }
+hex-literal = "0.4"
+k256 = { version = "=0.13.1", features = ["arithmetic", "serde", "expose-field", "std", "ecdsa"], default_features = false }
+
+[patch.crates-io]
+# Placing these patch statement in the workspace Cargo.toml will add RISC Zero SHA-256 and bigint
+# multiplicaation accelerator support for all downstream usages of the following crates.
+# DO NOT MERGE: Use tagged git references instead of branches
+sha2 = { git = "https://github.com/risc0/RustCrypto-hashes", tag = "sha2-v0.10.6-risc0" }
+k256 = { git = "https://github.com/risc0/RustCrypto-elliptic-curves", branch = "victor/use-risc0-accel"  }
```

Comment:
I don't think we should rely on `username/` branches persisting. So I suggest we rename these branches to something more stable and then updating those references here.

Reply:
Definitely. Strategy is to use tags primarily and have all of our tags live on the `risc0/use-accel` branch in our forks. I won't merge this PR until those tags are available.